properties/P000133.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: P000133
+slug: linearly_orderable
+name: Linearly orderable
+aliases:
+  - Linearly ordered topological space
+  - LOTS
```

Comment:
What do you think of the idea of making LOTS the primary name and making "Linearly orderable" the alias?  Would make it short and easily read in lists of theorems.